auslib/util/jsonschema_validators.py

```diff
@@ -24,6 +24,10 @@ def operator_validator(field_value):
     except TypeError:
         # get_op field returns None if no operator or no match, can't be unpacked
         raise jsonschema.ValidationError("Invalid input for buildID : %s. No Operator or Match found." % field_value)
+    try:
+        int(strip_operator(field_value))
+    except ValueError:
+        raise jsonschema.ValidationError("Invalid input for buildID: must be an integer.")
```

Comment:
It looks like the the very similar auslib.web.admin.views.forms.operator_validator() is now superceded by connexion, so a cleanup could be in order.